oauth2_provider/oauth2_validators.py

```diff
@@ -557,8 +557,12 @@ def rotate_refresh_token(self, request):
         """
         return oauth2_settings.ROTATE_REFRESH_TOKEN
 
-    @transaction.atomic
     def save_bearer_token(self, token, request, *args, **kwargs):
+        # Use the AccessToken's database instead of making the assumption it is in 'default'.
```

Comment:
Should we add a comment here that users who want to override the `save_bearer_token` behavior should override `_save_bearer_token_internals`?  Also lets call _save_bearer_token_internals to just _save_bearer_token... I find the _internals suffix confusing the _ prefer should be sufficient to indicate it is internal.

Reply:
I can do both. As for the name, yes, I was also uncertain. I renamed it to be very clear and catch people's attention. Looks like it worked.